src/bci_build/package/cosign/README.md.j2

```diff
@@ -0,0 +1,57 @@
+# {{ image.title }}
+{% include 'badges.j2' %}
+
+## Description
+Cosign aims to make signatures invisible infrastructure.
+
+Cosign supports:
+
+* "Keyless signing" with the Sigstore public good Fulcio certificate authority and Rekor transparency log (default)
+* Hardware and KMS signing
+* Signing with a cosign generated encrypted private/public keypair
+* Container Signing, Verification and Storage in an OCI registry.
+* Bring-your-own PKI
+
+
+## Usage
+
+### Verify a container image
+
+To verify the image, you'll need to pass in the expected certificate subject
+and certificate issuer via the `--certificate-identity` and
+`--certificate-oidc-issuer` flags:
+
+```shell
+$ podman run {{ image.pretty_reference }} \
+    verify $IMAGE \
+    --certificate-identity=$IDENTITY \
+    --certificate-oidc-issuer=$OIDC_ISSUER
+```
+
+You can also pass in a regex for the certificate identity and issuer flags,
+`--certificate-identity-regexp` and `--certificate-oidc-issuer-regexp`. See
+[Keyless verification using OpenID Connect](https://docs.sigstore.dev/cosign/verifying/verify/#keyless-verification-using-openid-connect)
+for details.
+
+### Verify a container image against a public key
+
+This command `verify` returns `0` if *at least one* `cosign` formatted signature for
+the image is found matching the public key. See the detailed usage below for
+information and caveats on other signature formats.
+
+Any valid payloads are printed to stdout, in json format. Note that these
+signed payloads include the digest of the container image, which is how we
+can be sure these "detached" signatures cover the correct image.
+
+```shell
+$ podman run {{ image.pretty_reference }} verify --key cosign.pub $IMAGE_URI:1h
+The following checks were performed on these signatures:
```

Comment:
```suggestion
The following checks are performed on the signatures:
```

Reply:
no, this is the literal command line output from upstream, we can't change that.

Reply:
I see! Then disregard my comment.